react-hs-examples/redux/stack.yaml

```diff
@@ -0,0 +1,11 @@
+resolver: lts-8.3
+packages:
+- .
+- location:
+    git: https://github.com/liqula/react-hs
+    commit: e6260f0162f1ced6dc03e54e2f1e24825369949a
+  extra-dep: true
+  subdirs:
+  - react-hs
```

Comment:
the other stack.yamls have deps into the local working copy, not the remote one.  if you want to change that, we should change it consistently all over.  but i like the way it is because your way makes it easy to accidentally keep references into older releases, which i would like to avoid.